### DISCUSSION TITLE
How to connect airflow and openmetadata

### DISCUSSION THREAD

**Discussion by b0kky** [open-metadata/OpenMetadata#16461] [Q&A] [answered]
Hello! Can someone explain me how i can connect my airflow server with openmetadata for run piplines? Which plugins will be installed? Documentation is not clever me for that. Thanks.
For begin i followed for that manual https://docs.open-metadata.org/v1.3.x/deployment/ingestion/openmetadata, and have some questions.
1.  Install the Connector Modules
```pip3 install "openmetadata-ingestion[<connector-name>]==x.y.z"```
it on opemnetadata side?
2. Install the Airflow APIs
is okay, install it on airflow side ```pip3 install "openmetadata-managed-apis==x.y.z"```
after installation got error  in airflow UI: 
`Broken plugin: [openmetadata_managed_apis.plugin] No module named 'metadata'`
What i'm doing wrong?
Openmetadata version 1.4
Airflow 2.8.4 (can be downgraded if need)

**Comment by Prajwal214** [comment, marked as answer]
@b0kky need to install the packages in the OM Server. Also need to install the connector module based on your requirement, you can use Airflow Version 2.7.x.

https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/setup.py

https://docs.open-metadata.org/v1.3.x/deployment/ingestion/openmetadata#custom-airflow-installation
